aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/MutationProcessor.java

```diff
@@ -85,20 +103,78 @@ void stopDrainingMutationOutbox() {
     }
 
     /**
-     * To process a StorageItemChange, we try to publish it to the remote GraphQL
-     * API. If that succeeds, then we can remove it from the outbox. Otherwise,
-     * we have to keep the mutation in the outbox, so that we can try to publish
-     * it again later, when network conditions become favorable again.
+     * Attempt to publish a change (update, delete, creation) over the network.
      * @param storageItemChange A storage item change to be published to remote API
-     * @return A single which completes with the successfully published item, or errors
+     * @return A single which completes with the successfully published item, or emits error
      *         if the publication fails
      */
-    @SuppressWarnings("checkstyle:MethodTypeParameterName") // The generics are complex, so use meaningful names
     private <MODEL extends Model, SIC extends StorageItemChange<MODEL>> Single<SIC> publishToNetwork(
```

Comment:
What's the idea behind using caps to distinguish between these two classes vs. a naming difference?

Reply:
I could use `<M extends Model, S extends StorageItemChange<M>>`, I guess.

Since there are so many generic types, and several types around here start with an `M` (`Model`, `ModelMetadata`, `ModelWithMetadata`, etc.) I thought it'd be clearer to read if I was explicit with the `MODEL`, etc.

Reply:
Ok, what's the difference between MODEL and Model? Would it make sense to give MODEL a name that reflects that difference?

Reply:
Update: I got rid of all this crap, in a subsequent update. It ends up being easier to read with just the `<T extends Model>` that we use eery where else.